spec/controllers/spree/admin/images_controller_spec.rb

```diff
@@ -18,6 +18,33 @@ module Admin
           expect(assigns(:image).variant_ids).to include(master.id)
         end
       end
+
+      describe 'POST #create' do
+        let(:variant) { create(:variant) }
+        let(:product) { variant.product }
+
+        let(:create_path) {
+          "/admin/products/#{product.id}/images"
+        }
+        let(:create_params) {
+          {
+            image: {
+              viewable_ids: [variant.id],
+            },
+          }
+        }
+
+        before do
```

Comment:
This should probably be an `around` so we can restore its value, no?

Reply:
Yep, done.